src/lamplib/src/genny/tasks/preprocess.py

```diff
@@ -381,6 +388,29 @@ def _replace_formatstr(self, input):
             )
             raise ParseException(msg)
 
+    def _replace_clienturi(self, input):
+        OP_KEY = "^ClientURI"
+
+        name: Optional[str] = input.get("Name", None)
```

Comment:
`Optional` does not appear to have been imported.

Reply:
ahh rip haha, good catch, bad merge conflict 😭